CMakeLists.txt

```
@@ -1,12 +1,10 @@
-cmake_minimum_required(VERSION 3.10...3.16)
-if(${CMAKE_VERSION} VERSION_LESS 3.12)
-	cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
-endif()
+cmake_minimum_required(VERSION 3.13...3.19)
 
 project(ddsim
-        VERSION 1.0.2
+		LANGUAGES CXX
+        VERSION 1.1.0
         DESCRIPTION "A front end to utilize the dd_package and qfr for simulation"
```

Shouldn't the description be more in line with the README "header"? Something along the lines of 
```suggestion
        DESCRIPTION "DDSIM - A JKQ quantum simulator based on decision diagrams"
```